hconfig_utils/mapl3hconfig_get.F90

```diff
@@ -0,0 +1,10 @@
+module mapl3hconfig_get
```

Comment:
The prefix being used for mapl3 is `mapl3g_`
```suggestion
module mapl3g_hconfig_get
```

Reply:
will do